free-2.md

```diff
@@ -1,44 +1,57 @@
-# Free project 2
+# Free project 2 -- Wine pairings
 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would help wine enthusiasts choose wines that pair well with the food that they are eating or making. A language would be nice because wine pairings seem like something of a closed set of terms, which could be implemented in various ways.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Lots of people like wine and food and consider themselves to be foodies. However, lots of people probably don't know be heart exactly which foods and wines go well togther, and want to improve their eating and drinking experiences. I could imagine a person who is eating a steak with potatoes, and wants to pair a wine with it but isn't sure which wine they want; perhaps they could even input to the program a list of wines that they are choosing between and the program/language could then help them choose the most appropriate choice. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL seems like a good choice because wine pairing is a fairly narrow domain, and users could use the language in order to educate themselves without having to go through too much other information. A language devoted strictly to this topic could be really helpful in assisting users expand their tastes for wine and food.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+My whole family are huge wine snobs, yet I find that I am often lacking in the taste refinement that they have. I feel like I would totally use an app like this to improve my own experiences with food and wine, and I feel like even experienced wine drinkers could learn some new things about the way they experience wine.
```

Comment:
A similar comment to above. How does this different from the numerous wine apps already on the Android Market (or App Store)?